web/src/components/admin/connectors/ConnectorForm.tsx

```diff
@@ -144,6 +146,7 @@ export function ConnectorForm<T extends Yup.AnyObject>({
             input_type: inputType,
             connector_specific_config: connectorConfig,
             refresh_freq: refreshFreq || 0,
+            prune_freq: pruneFreq || null,
```

Comment:
if `pruneFreq` is 0, this will set it to `null` which is not intended I believe?